src/brouwers/kitreviews/models.py

```diff
@@ -108,5 +104,23 @@ class Meta:
 
     def __unicode__(self):
         return _(u"Vote for review by %(review_submitter)s") % {
-                    'review_submitter': get_username(self.kit_review, field='reviewer')
-                    }
+            'review_submitter': get_username(self.kit_review, field='reviewer')
```

Comment:
`get_username` is broken iirc, you can just use `self.kit_review.reviewer.username`.